HyperVExtension/src/HyperVExtension/Helpers/BytesHelper.cs

```diff
@@ -30,7 +33,46 @@ public static string ConvertFromBytes(ulong size)
         {
             return $"{(size / OneGbInBytes).ToString("F", CultureInfo.InvariantCulture)} GB";
         }
+        else if (size >= OneMbInBytes)
+        {
+            return $"{(size / OneMbInBytes).ToString("F", CultureInfo.InvariantCulture)} MB";
+        }
 
-        return $"{(size / OneMbInBytes).ToString("F", CultureInfo.InvariantCulture)} MB";
+        return $"{(size / OneKbInBytes).ToString("F", CultureInfo.InvariantCulture)} KB";
+    }
+
+    /// <summary>
+    /// Converts from a ulong amount of bytes to a localized string representation of that byte size in gigabytes.
+    /// Relies on StrFormatByteSizeEx to convert to localized.
+    /// </summary>
+    /// <returns>
+    /// If succeeds internally return localized size in gigabytes, otherwise falls back to community toolkit
+    /// implementation which is in English.
+    /// </returns>
+    public static string ConvertBytesToString(ulong sizeInBytes)
+    {
+        unsafe
+        {
+            // 15 characters + null terminator.
+            var buffer = new string(' ', 16);
+            fixed (char* tempPath = buffer)
+            {
+                var result =
+                    PInvoke.StrFormatByteSizeEx(
+                        sizeInBytes,
+                        SFBS_FLAGS.SFBS_FLAGS_TRUNCATE_UNDISPLAYED_DECIMAL_DIGITS,
+                        tempPath,
+                        PInvoke.MAX_PATH);
+                if (result != 0)

```

Comment:
HRESULT.S_OK can be used instead of 0. (S_OK is already in NativeMethods.txt)

Reply:
Thanks updated